previews/partial-reruns/Home.py

```diff
@@ -53,6 +53,21 @@ def simple_chart():
   This is great for streaming or polling chart updates, or updating progress on a long-running background task.
 """
 
+with st.expander("👀 More details about `@st.partial` behavior"):
```

Comment:
@sfc-gh-dmatthews @vdonato if you have any feedback on particularly this section, feel free to add and I'll address it in another PR